app/assets/stylesheets/practices.scss

```diff
@@ -1,3 +1,50 @@
-// Place all the styles related to the Practices controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+h3 {
+  margin: 10px;
+  font-weight: 900;
+}
+
+.profile {
+  background-color: #fff;
+  border-bottom: 1px solid #ddd;
+  margin: none;
+  padding: 10px;
+}
+
+.practices-list {
+  list-style-type: none;
+  margin: 20px 5px;
+  .practice {
+    background-color: #fff;
+    border: 1px solid #ddd;
+    border-radius: 3px;
+  }
+}
+
+.practice {
+  .title {
+    border-bottom: 1px solid #ddd;
+    margin: none;
+    .fa.fa-check-square-o {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3